Undefined name 'uri' in smart_open_lib.py

flake8 testing of https://github.com/RaRe-Technologies/smart_open on Python 3.7.0

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./smart_open/smart_open_lib.py:438:53: F821 undefined name 'uri'
        raise RuntimeError("invalid HDFS URI: %s" % uri)
                                                    ^
./smart_open/smart_open_lib.py:448:56: F821 undefined name 'uri'
        raise RuntimeError("invalid WebHDFS URI: %s" % uri)
                                                       ^
./smart_open/smart_open_lib.py:505:53: F821 undefined name 'uri'
        raise RuntimeError("invalid file URI: %s" % uri)
                                                    ^
3     F821 undefined name 'uri'
3
```

--- a/smart_open/smart_open_lib.py
+++ b/smart_open/smart_open_lib.py
@@ -435,7 +435,7 @@ def _parse_uri_hdfs(parsed_uri):
     uri_path = parsed_uri.netloc + parsed_uri.path
     uri_path = "/" + uri_path.lstrip("/")
     if not uri_path:
-        raise RuntimeError("invalid HDFS URI: %s" % uri)
+        raise RuntimeError("invalid HDFS URI: %s" % parsed_uri)
     return Uri(scheme='hdfs', uri_path=uri_path)
 
 
@@ -445,7 +445,7 @@ def _parse_uri_webhdfs(parsed_uri):
     if parsed_uri.query:
         uri_path += "?" + parsed_uri.query
     if not uri_path:
-        raise RuntimeError("invalid WebHDFS URI: %s" % uri)
+        raise RuntimeError("invalid WebHDFS URI: %s" % parsed_uri)
     return Uri(scheme='webhdfs', uri_path=uri_path)
 
 
@@ -502,7 +502,7 @@ def _parse_uri_file(parsed_uri):
     uri_path = os.path.expanduser(uri_path)
 
     if not uri_path:
-        raise RuntimeError("invalid file URI: %s" % uri)
+        raise RuntimeError("invalid file URI: %s" % parsed_uri)
 
     return Uri(scheme='file', uri_path=uri_path)
 

